presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -96,6 +96,8 @@
 
     private List<String> resourceConfigFiles = ImmutableList.of();
 
+    private DataSize maxLineLength = new DataSize(100, DataSize.Unit.MEGABYTE);
```

Comment:
Let's name this and related `textMaxLineLength` so that it's clear this is for text.